web/templates/keyboard_shortcuts.hbs

```diff
@@ -171,6 +171,14 @@
                     <td class="definition">{{t 'Cycle between channel views' }}</td>
                     <td><span class="hotkey"><kbd>Shift</kbd> + <kbd>A</kbd> or <kbd>Shift</kbd> + <kbd>D</kbd></span></td>
                 </tr>
+                <tr>
+                    <td class="definition">{{t 'Go to inbox' }}</td>
+                    <td><span class="hotkey"><kbd>Shift</kbd> + <kbd>I</kbd></span></td>
+                </tr>
+                <tr>
+                    <td class="definition">{{t 'Go to recent conversations' }}</td>
```

Comment:
This duplicates `T` from the Recent conversations area of the keyboard-shortcut overlay, where we've got different language:

![Screenshot 2024-09-25 at 10 01 59 AM](https://github.com/user-attachments/assets/1fad4c0c-cab4-4d91-ae32-7dbae66d10cc)

Not sure if that should be changed to "Go to recent conversations," too.

Reply:
Hmm fair point :thinking: . From my admittedly newbie perspective I see two different conventions:

- In the navigation section, mostly "Go to X" phrasing
- In the dedicated view-specific sections below, language like "View recent conversations" or "View drafts"

I'm cool with making more changes or leaving this as is. (I'll default to _not_ changing more stuff for the moment though.)

Reply:
I noted on the CZO thread:

> The Recent conversations one will be duplicative with the dedicated section, but that's fine.